ministryofjustice/laa-apply-for-criminal-legal-aid
# Add housing payment type page

## Description of change
Add housing payment type page 

## Link to relevant ticket
https://dsdmoj.atlassian.net/browse/CRIMAPP-274
## Notes for reviewer

## Screenshots of changes (if applicable)

### After changes:
<img width="957" alt="image" src="https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/assets/45827968/0352f7c7-6bbf-4aa8-828b-60afee1554d6">

## How to manually test the feature

1. rails db:migrate
2. Start an application
3. Go to steps/outgoings//housing_payments_where_client_pays
4. Check that submitting the form without selecting an option displays error 'Select which payment your client pays where they usually live"
5. Check that submitting with an option selected submits ok
